iset.mm

```diff
@@ -21146,14 +21274,27 @@ effectively bound by a quantifier or something that expands to one (such
   nfra1 $p |- F/ x A. x e. A ph $=
     ( wral cv wcel wi wal df-ral nfa1 nfxfr ) ABCDBECFAGZBHBABCILBJK $.
 
+  ${
+    $d x y $.
+    nfraldw.1 $e |- F/ y ph $.
+    nfraldw.2 $e |- ( ph -> F/_ x A ) $.
+    nfraldw.3 $e |- ( ph -> F/ x ps ) $.
+    $( Not-free for restricted universal quantification where ` x ` and ` y `
+       are distinct.  See ~ nfraldya for a version with ` y ` and ` A `
+       distinct instead.  (Contributed by NM, 15-Feb-2013.)  (Revised by Gino
+       Giotto, 10-Jan-2024.) $)
+    nfraldw $p |- ( ph -> F/ x A. y e. A ps ) $=
+      ( wral cv wcel wi wal df-ral nfcvd nfeld nfimd nfald nfxfrd ) BDEIDJZEKZB
+      LZDMACBDENAUBCDFAUABCACTEACTOGPHQRS $.
+  $}
+
   ${
     $d x y $.
     nfraldxy.2 $e |- F/ y ph $.
     nfraldxy.3 $e |- ( ph -> F/_ x A ) $.
     nfraldxy.4 $e |- ( ph -> F/ x ps ) $.
-    $( Not-free for restricted universal quantification where ` x ` and ` y `
-       are distinct.  See ~ nfraldya for a version with ` y ` and ` A `
-       distinct instead.  (Contributed by Jim Kingdon, 29-May-2018.) $)
+    $( Old name for ~ nfraldw .  (Contributed by Jim Kingdon, 29-May-2018.)
+       (New usage is discouraged.) $)
     nfraldxy $p |- ( ph -> F/ x A. y e. A ps ) $=
       ( wral cv wcel wi wal df-ral wnfc nfcv a1i nfeld nfimd nfald nfxfrd ) BDE
       IDJZEKZBLZDMACBDENAUDCDFAUCBCACUBECUBOACUBPQGRHSTUA $.
```

Comment:
Why not remove it ?

Reply:
Well, that it will get removed is the idea (probably by me down the road, or if someone else gets to it first, so much the better).

Trying to pull on that thread in this pull request seemed like a little more than I wanted to take on, given that I started pulling on intuitionizing fprod2d and something like a dozen predicate logic theorems came off in my hand.

An ancillary benefit is that anyone else who works on iset.mm will get a chance to update their branches and habits and whatever else they might have lying around.

Reply:
@jkingdon When will it get removed ? What should we wait ?
* Is it a date ? In which case, you can write
  > "Obsolete as of 00-xxx-2024. "
  
  so that we know that starting from one year later, anyone can remove it.

* Or is it something else ? In which case, you can write 
  > TODO: check that blablabla, or do blablabla and then remove this theorem.

I'm not asking that you or anyone specific does it, but simply that we know what should be done and when one can do it without risking conflicting with someone else's work.

This kind of thing is necessary as soon as more than one person is working on a project, to avoid miscommunication, or that everyone forget why in the first place we have an old name and what we should do about it.

(Personal note: even in some personal projects, I do as if "more than one person" work on it, because I know that when coming back to it after a pause, I will probably have forgotten some loose ends.)

Reply:
It will probably be removed once its 40 usages are removed

Reply:
Then, its comment should contain:

> TODO: replace its usages by ~nfraldw and then remove.

so that this information is shared among the contributors. If you say to yourself "it's on my TODO list, I'll do it then", then other contributors are left wondering, they may engage in parallel work which may end up being wasted time, etc.

Beyond this particular case (which, now, looks admittedly benign), this is more about a general practice for collaborative projects, that I'm advocating.